### PR TITLE
NR-86000 Updated Blazor Server text changes.

### PR DESCRIPTION
**Pull request by pkudikyala**
# Summary

**Ticket Url:** https://issues.newrelic.com/browse/NR-86000
Description: Updated on text changes to the Blazor Server IO page as mentioned in the [PR1535](https://github.com/newrelic/newrelic-quickstarts/pull/1535)

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Are all documentation links publically accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ ] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### Screenshots:

screencapture-newrelic-instant-observability-preview-2023-02-13-11_13_25

### CODE REVIEW

**Review by pkosanam — APPROVED** [review]
looks good

File: quickstarts/dotnet/blazor-server/config.yml [hunk 1]
```diff
@@ -2,27 +2,26 @@ id: b038e759-bf74-4353-9273-c66e86b25ebe
...
-  Blazor Server offers a lot of useful features in addition to allowing C# developers to utilize their preferred language for web development. Blazor Server allows you to run your code through Blazor Server on the server or the client, and you can decide which users or devices experience each method.
-  
-  There are a few compelling arguments that support the employment of Blazor Server:
-  - All of the major browsers support Blazor Server.
-  - Using C# for interactive web apps.
-  - Utilize current C# libraries.
+  ## Why monitor your Blazor Server?
+  There are a few strong arguments in favour of using Blazor Server:
+
+  - Monitoring the Higher Letency and Slower Application.
```
Comment (brammerl) [inline quickstarts/dotnet/blazor-server/config.yml:11]:
```suggestion
  - Monitoring the Higher Latency and Slower Application.
```

Comment (pkudikyala) [reply quickstarts/dotnet/blazor-server/config.yml]: Hi @brammerl ,

File: quickstarts/dotnet/blazor-server/config.yml [hunk 2]
```diff
@@ -2,27 +2,26 @@ id: b038e759-bf74-4353-9273-c66e86b25ebe
...
 
-  ## Why monitor Blazor Server?
-  Blazor Server offers a lot of useful features in addition to allowing C# developers to utilize their preferred language for web development. Blazor Server allows you to run your code through Blazor Server on the server or the client, and you can decide which users or devices experience each method.
-  
-  There are a few compelling arguments that support the employment of Blazor Server:
-  - All of the major browsers support Blazor Server.
-  - Using C# for interactive web apps.
-  - Utilize current C# libraries.
+  ## Why monitor your Blazor Server?
+  There are a few strong arguments in favour of using Blazor Server:
```
[the remark this answers is not in the document]
Comment (pkudikyala) [reply quickstarts/dotnet/blazor-server/config.yml]: Hi @brammerl ,
Updated text changes, Kindly review it.

Thank you...

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**